src/ar_weave.erl

```diff
@@ -351,20 +352,3 @@ read_genesis_txs() ->
 		[],
 		Files
 	).
-
-create_genesis_txs() ->
```

Comment:
I think it makes sense to keep this code even though we're not planning to use it again. Did you get Sam's opinion on this? I think it makes more sense to remove the data than removing the code, so maybe we should remove the `data/genesis_wallets.csv` file or truncate it (into only one line with the column names).